README.md

```diff
@@ -1,5 +1,13 @@
 # VGA Simulator
 
+**Note:** This project is not actively maintained. Consider using one of the following newer ports:
+
+- [Pedro, @pvieito](https://github.com/pvieito) made a CLI version of this app in Python that you can find in [this gist](https://gist.github.com/pvieito/8cdb54a9a03fd36e51c8df6e331a3006).
+- https://github.com/ARC-Lab-UF/vga-simulator is an actively maintained port of Pedro's Python version, with a beautiful and flexible command line interface.
```

Comment:
> is an actively maintained port of Pedro's Python version, with a beautiful and flexible command line interface.

This description will fall out of date (especially the  "actively maintained" part. I'm not sure

We can probably describe this a better way as "beautiful and flexible command line interface" is kinda odd to read and all of the positional parameters don't look easy to get right.

Reply:
That's fair. I think I was trying to market it a little bit lol. I'll simplify the description.

Reply:
Feel free to post an issue if you have any ideas on how to improve the CLI 😄